code/modules/mob/living/carbon/xenomorph/Castes/Sentinel.dm

```diff
@@ -80,18 +80,18 @@
 	upgrade = 2
 
 	// *** Melee Attacks *** //
-	melee_damage_lower = 25
-	melee_damage_upper = 30
+	melee_damage_lower = 23
```

Comment:
Here as well. A "tweaked the values of this and that caste, buffing this and nerfing that" wouldn't hurt.